webapp/src/main/java/org/apache/atlas/notification/EntityNotificationListenerV2.java

```diff
@@ -191,6 +193,27 @@ private void notifyClassificationEvents(List<AtlasEntity> entities, OperationTyp
         RequestContext.get().endMetricRecord(metric);
     }
 
+    private void notifyClassificationEvents(List<AtlasEntity> entities, OperationType operationType, Object mutatedObj, boolean forceInline) throws AtlasBaseException {
```

Comment:
Call this method in this method "private void notifyClassificationEvents(List<AtlasEntity> entities, OperationType operationType, Object mutatedObj)" here